src/index.ts

```diff
@@ -327,6 +327,68 @@ export class FlashbotsBundleProvider extends providers.JsonRpcProvider {
     return this.sendRawBundle(signedTransactions, targetBlockNumber, opts)
   }
 
+  public async sendPrivateTransaction(
+    transaction: FlashbotsBundleTransaction,
+    opts?: {
+      targetBlockNumber?: number,
+      maxBlockNumber?: number,
+      minTimestamp?: number,
+    },
+  ): Promise<FlashbotsTransaction> {
+    return this.sendPrivateRawTransaction(
+      await transaction.signer.signTransaction(transaction.transaction),
+      opts,
+    )
+  }
+
+  public async sendPrivateRawTransaction(
+    signedTransaction: string,
+    opts?: {
+      targetBlockNumber?: number, // block to check for tx inclusion using wait()
```

Comment:
I don't really like this. The `wait` function relies on a "target block number" to check for bundle inclusion, but that really only makes sense for bundles. I think we need a different "wait" function for the sendPrivateTransaction endpoint, which checks each block for tx inclusion up to maxBlockNumber and returns if/when it finds the tx included in a block.

Thoughts?

Reply:
I threw out `targetBlockNumber` for this function. It uses the `maxBlockNumber` param to tell the block event listener how many blocks in which to search for the tx's inclusion. Simulations (via the `simulation` callback) are done at the block specified by the `startBlockNumber` param.